src/mrinufft/trajectories/utils.py

```diff
@@ -144,6 +150,27 @@ class Packings(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     SPIRAL = FIBONACCI
 
 
+#############################
+# Variable Density Sampling #
+#############################
+
+
+class VDSorder(StrEnum):
+    """Available ordering for variable density sampling."""
+
+    CENTER_OUT = "center-out"
+    RANDOM = "random"
+    TOP_DOWN = "top-down"
```

Comment:
It could be great at some point (not here) to have tools to manipulate the shot order